apps/accounts/models.py

```diff
@@ -238,8 +238,8 @@ def confirmed_email(self):
         else:
             confirmed_emails = self.get_confirmed_emails()
             if confirmed_emails.exists():
-
-                email = confirmed_emails.first().email
+                confirmed_emails = confirmed_emails.first()
```

Comment:
This shouldn't be necessary, because this is already inside of `if confirmed_emails.exists()`

Reply:
Ok. But not sure how `confirmed_emails.exists()` succeeded and failed to get `confirmed_emails.first().email`. We can close this PR if not needed.